recipe/meta.yaml

```diff
@@ -6,39 +6,53 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/googlemaps-{{ version }}.tar.gz
-  sha256: 3055fcbb1aa262a9159b589b5e6af762b10e80634ae11c59495bd44867e47d88
+  url: https://github.com/googlemaps/google-maps-services-python/archive/refs/tags/v4.10.0.tar.gz
```

Comment:
```suggestion
  # use GH release archive for tests
  url: https://github.com/googlemaps/google-maps-services-python/archive/refs/tags/v{{ version }}.tar.gz
```

Reply:
note the `{{ version }}` in the URL, and we need a comment for why we switch to GH release archive because our standard is to source from PyPi